jenkins/osa_onmetal.functions.groovy

```diff
@@ -205,7 +205,51 @@ def delete_onmetal(datacenter_tag) {
 
 }
 
+def setup_during_test(host_ip) {
```

Comment:
I'd prefer if this function is called something like install_during_upgrade_tests so it is consistent with the other upgrade tests suites like install_persistent_resources_tests